hathor/cli/top.py

```diff
@@ -26,6 +24,12 @@
 from math import floor
 from typing import Any, Callable, DefaultDict, Dict, List, Optional, Tuple
 
+try:
+    import curses
+    import curses.ascii
+except ModuleNotFoundError:
+    pass
```

Comment:
Shouldn't we print a friendly error message and exit?

Reply:
If we exit here this can't even be imported on Windows. The obvious alternative would be to import inside the methods that use it, they there are a lot. I think it's a good compromise to let it fail with `curses undefined` when it is being used, than to fail immediately or give a warning when most of the time this isn't what you will be running.

This mostly happens because every subcommand module on `hathor/cli` is imported before any subcommand is selected.

Reply:
You're right. And the alternative is bad because it is used everywhere in the code.

Ok. I agree this is the best method for now.